DB Setup/createDB.sql

```diff
@@ -10,6 +10,7 @@ create table SessionKeys (
 
 create table tfiSurvey ( 
 	tfiSurveyID int NOT NULL, 
+        Test_Taken bool, 
```

Comment:
It looks like a bunch of random lines are indented in this file. Is there a reason for this or can everything be lined up?

Reply:
I don't know why it did that. I tried fixing it before but i just changed the indentations on the whole file so hopefully its fixed now.